### DISCUSSION TITLE
Build of Grapesjs

### DISCUSSION THREAD

**Discussion by siddhi1397** [GrapesJS/grapesjs#5498] [Q&A]
How to create build of grapesjs in Angular after doing few changes in the library?
![image](https://github.com/GrapesJS/grapesjs/assets/121800257/cdc887bd-2f09-4701-91ac-9f51037a4b39)

**Comment by artf** [comment]
Looks like you're trying to build it from the distributed package via npm. The proper way would be to clone the repo and build it from the source.

**Reply by siddhi1397** [reply]
Thank you for the help
